zebrad/tests/common/get_block_template_rpcs/submit_block.rs

```diff
@@ -0,0 +1,161 @@
+use std::path::PathBuf;
```

Comment:
Please add a module doc comment to new modules.

Reply:
Updated in [`4959a78`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/4959a78f3c206192ec402f65b29fa5e39ebb9ec2)